File: tier1/kotlin/3bbbebd9/Store.kt
package app

import java.sql.DriverManager

fun lookup(code: String): String {
    val connection = DriverManager.getConnection("jdbc:h2:mem:app")
    val statement = connection.createStatement()
    val results = statement.executeQuery("SELECT status FROM orders WHERE code = '" + code + "'")
    return if (results.next()) results.getString(1) else ""
}

File: tier1/go/13ce88dd/store.go
package app

import (
	"database/sql"
)

var db *sql.DB

func Lookup(code string) (string, error) {
	statement := "SELECT status FROM orders WHERE code = $1"
	var status string
	err := db.QueryRow(statement, code).Scan(&status)
	return status, err
}
